src/components/implementation/no_interface/llbooter/Makefile

```diff
@@ -4,7 +4,7 @@ COMPONENT=llboot_test.o
 INTERFACES=
 DEPENDENCIES=
 IF_LIB=
-ADDITIONAL_LIBS=-lcobj_format -lcos_kernel_api
+ADDITIONAL_LIBS=-lcobj_format -lcos_kernel_api -lcos_defkernel_api -lsl -lheap -lsl_thd_static_backend -lsl_lock -lsl_mod_fprr
```

Comment:
This includes @maloneya PR which was never merged because of this reason. I can fix this in a commit that comes after the other changes I'm making right now.